packages/calcite-components/src/components/date-picker/date-picker.e2e.ts

```diff
@@ -246,8 +247,8 @@ describe("calcite-date-picker", () => {
       element.setProperty("max", undefined);
       await page.waitForChanges();
 
-      expect(await element.getProperty("minAsDate")).toBe(undefined);
-      expect(await element.getProperty("maxAsDate")).toBe(undefined);
+      expect(await element.getProperty("minAsDate")).toMatchObject({});
```

Comment:
FIXME: it looks like these changes prevent unsetting `min` and `max`, which we still need to allow.